Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -414,6 +464,69 @@ import BraintreeDataCollector
         }
     }
 
+    private func edit(request: BTPayPalVaultEditRequest, completion: @escaping (BTPayPalVaultEditResult?, Error?) -> Void
+    ) {
+        apiClient.fetchOrReturnRemoteConfiguration { configuration, error in
+            if let error {
+                self.notifyEditFIFailure(with: error, completion: completion)
+                return
+            }
+
+            guard let configuration, let json = configuration.json else {
+                self.notifyEditFIFailure(with: BTPayPalError.fetchConfigurationFailed, completion: completion)
+                return
+            }
+
+            self.isConfigFromCache = configuration.isFromCache
+
+            guard json["paypalEnabled"].isTrue else {
+                self.notifyEditFIFailure(with: BTPayPalError.disabled, completion: completion)
+                return
+            }
+
+            let dataCollector = BTDataCollector(apiClient: self.apiClient)
+
+            var riskCorrelationID: String
+            var parameters: [String: Any]
+
+            if let editRequest = request as? BTPayPalVaultErrorHandlingEditRequest {
+                riskCorrelationID = dataCollector.clientMetadataID(editRequest.riskCorrelationID)
+                parameters = editRequest.parameters()
```

Comment:
Should we be passing the `riskCorrelationID` here?

Reply:
That was what I understood from @sarahkoop

Reply:
I don't think so, I think I caused confusion with unnecessary call to DataCollector.clientMetadataID function there.
We are just using raw riskCorrelationID passed into the request there.

Reply:
I was wrong about what Sarah told us - she told us to use CMID function with the merchant passed in riskCorrelationID. 
I've made corrections on this commit e5f98e80d94363d2b6e0307b06ea721695e17004 
to use the data collector CMID function for error handling flow with merchant passed in value.

I've posted question on collaboration channel just to double check on
1. whether we should be passing in riskCorrelationID for both edit flows
2. whether both flows should be using the data collector CMID function to generate this value passed into gateway